packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/ProjectCard.js

```diff
@@ -1,117 +1,166 @@
 /**
- * Tupaia Web
- * Copyright (c) 2019 Beyond Essential Systems Pty Ltd.
- * This source code is licensed under the AGPL-3.0 license
- * found in the LICENSE file in the root directory of this source tree.
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
-
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { BOX_SHADOW, WHITE, LIGHT_GREY } from '../../../../styles';
+import Lock from '@material-ui/icons/Lock';
+import { darken } from '@material-ui/core/styles';
+import Alarm from '@material-ui/icons/Alarm';
+import { Button, Typography } from '@material-ui/core';
 
 const Card = styled.div`
-  display: grid;
-  grid-template-columns: auto;
-  grid-template-rows: auto auto auto;
-  gap: 16px;
-  padding-bottom: 16px;
-  border-radius: 3px;
-  text-align: center;
-  position: relative;
-  box-shadow: ${BOX_SHADOW};
-  background: ${WHITE};
-  color: #000000;
-`;
-
-const Title = styled.div`
-  font-size: 22px;
-  font-weight: 500;
-  margin-top: -8px;
-  padding: 0 16px;
-`;
-
-const Header = styled.div`
-  position: relative;
   display: flex;
-  justify-content: center;
-  margin-bottom: 15px;
-  height: 120px;
-  background-color: ${LIGHT_GREY}; /* fallback color */
-  background-image: ${({ backgroundImage }) => `url(${backgroundImage})`};
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
+  flex-direction: column;
+  padding: 2.5rem;
+  border-radius: 5px;
+  background: #2e2f33;
```

Comment:
The colors in these components are not used anywhere else so I didn't make them consts.